snapcraft/plugins/kbuild.py

```diff
@@ -103,13 +109,24 @@ def schema(cls):
             'default': None,
         }
 
+        schema['properties']['build-attributes'] = {
```

Comment:
this is already in the main schema and not really needed here (in `schema/snapcraft.yaml`)